parmed/tools/add1264.py

```diff
@@ -78,89 +78,88 @@ def params1264(parm, mask, c4file, watermodel, polfile, tunfactor):
 
     print("***********************************************************")
     # Determine which atom type was treated as the center metal ion
-    typelist = dict()
+    mettypdict = dict()
     for i in mask.Selected():
-        typeidx = parm.parm_data['ATOM_TYPE_INDEX'][i]
-        chgi = parm.parm_data['CHARGE'][i]
-        if typeidx in typelist: continue
-        typelist[typeidx] = (parm.atoms[i].atomic_number, int(chgi))
-        print("The selected metal ion is",
-              pt.Element[parm.atoms[i].atomic_number])
-    types = typelist.keys()
-    types.sort()
-
-    # Determine the C4 term between the atom type of center metal ion 
-    # and every atom type in the prmtop file 
-
-    # 1. Get the list of AMBER_ATOM_TYPE and ATOM_TYPE_INDEX for all 
+        mettypind = parm.parm_data['ATOM_TYPE_INDEX'][i]
+        metchg = parm.parm_data['CHARGE'][i]
+        if mettypind in mettypdict: continue
+        mettypdict[mettypind] = (parm.atoms[i].atomic_number, int(metchg))
+        print "The selected metal ion is", \
+                pt.Element[parm.atoms[i].atomic_number]
```

Comment:
This needs to be changed to:

```Python
        print("The selected metal ion is",
              pt.Element[parm.atoms[i].atomic_number])
```

We support both Python 2 and Python 3 in the same code base, and this won't work with Python 3.  You can put `from __future__ import print_function` at the top to get the right behavior.